### ISSUE TITLE
ios / android with screenreader has issue

### ISSUE THREAD

**Issue by ghost** [highcharts/highcharts#10019]
#### Expected behaviour
1. When screen reader is on the chart (eg: aria) should display focused element (point on series) on chart.
2. When screen reader is on when we swipe right on mobile (android) the screen reader should read the point description and tooltip should be displayed at that point.
3. When screen reader is on when the focus goes on chart on mobile (eg: Iphone 7) at one time the series is displayed as outlined (should not focus on series line)
4. When screen reader is on and when we swipe right on mobile (Iphone 7) focused on chart series the focus point  should display from x=0 and y=0

#### Actual behaviour
1. When screen reader is on the chart (eg: aria) doen't display focus on series chart when we swipe.
2. When screen reader is on when we swipe right on mobile (android) the screen reader reads the point description but no tooltip is displayed on that point.
3. When screen reader is on when the focus goes on chart on mobile (eg: Iphone 7) at one time the series is displayed as outlined which is at some weird position (not displayed where the exact series line)
4. When screen reader is on and when we swipe right on mobile (Iphone 7) focused on chart series the point starts below the chart (not from where the x and y axis starts)

#### Live demo with steps to reproduce

https://www.highcharts.com/android/demo
https://www.highcharts.com/ios/demo
#### Product version

Highchart version 7.0.2
#### Affected browser(s)
android / Iphone : safari , chrome

**Comment by SudeepSinghGusain** [comment]
It is reproduce-able at : 
https://www.highcharts.com/android/demo
https://www.highcharts.com/ios/demo

**Comment by KacperMadej** [comment]
Hi @SudeepSinghGusain 

Thank you for reporting about this issue.

FYI: @oysteinmoseng   accessibility problem

**Comment by SudeepSinghGusain** [comment]
Also on moline when screen reader is on user is not able to exit series. It loops again and again in the series.
